addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -0,0 +1,323 @@
+odoo.define('survey.session_manage', function (require) {
+'use strict';
+
+var publicWidget = require('web.public.widget');
+var core = require('web.core');
+var _t = core._t;
+
+publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend({
+    selector: '.o_survey_session_manage',
+    events: {
+        'click .o_survey_session_copy': '_onCopySessionLink',
+        'click .o_survey_session_next, .o_survey_session_start': '_onNextQuestionClick',
+        'click .o_survey_session_end': '_onEndSessionClick',
+        'click .o_survey_session_results_href': '_onResultsTabClick',
+        'click .o_survey_session_ranking_href': '_onRankingTabClick',
+        'click .o_survey_session_refresh_results': '_onRefreshResultsClick',
+        'click .o_survey_session_refresh_ranking': '_onRefreshRankingClick',
+    },
+
+    /**
+     * Overridden to set a few properties that come from the python template rendering.
+     *
+     * We also handle the timer IF we're not "transitioning", meaning a fade out of the previous
```

Comment:
just add a note to tell it is managed by the isRpcCall variable (if I understood well)